lib/dgram.js

```diff
@@ -256,11 +257,21 @@ Socket.prototype.sendto = function(buffer,
                                    port,
                                    address,
                                    callback) {
-  if (typeof offset !== 'number' || typeof length !== 'number')
-    throw new Error('Send takes "offset" and "length" as args 2 and 3');
+  if (typeof offset !== 'number') {
+    throw new errors.Error('ERR_INVALID_ARG_TYPE', 'offset', 'number');
```

Comment:
I help off changing the types of errors as that may be more breaking than just changing the message. @jasnell what's your take on that front ?